platform/src/main/java/arm/org/tron/common/math/MathWrapper.java

```diff
@@ -0,0 +1,252 @@
+package org.tron.common.math;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * This class is deprecated and should not be used in new code,
+ * for cross-platform consistency, please use {@link StrictMathWrapper} instead,
+ * especially for floating-point calculations.
+ */
+@Deprecated
+public class MathWrapper {
+
+  private static final Map<PowData, Double> powData = Collections.synchronizedMap(new HashMap<>());
+  private static final String EXPONENT = "3f40624dd2f1a9fc"; // 1/2000 = 0.0005
+
+  public static double pow(double a, double b) {
+    double strictResult = StrictMath.pow(a, b);
+    return powData.getOrDefault(new PowData(a, b), strictResult);
+  }
+
+  /**
+   * This static block is used to initialize the data map.
+   */
+  static {
+    addPowData("3ff0192278704be3", EXPONENT, "3ff000033518c576"); //  4137160
```

Comment:
May optimze the first comment:  block 4137160 of mainnet. Otherwise it's confusing.